app/export.go

```diff
@@ -110,14 +113,24 @@ func (app *App) prepForZeroHeightGenesis(ctx sdk.Context, jailAllowedAddrs []str
 		feePool.CommunityPool = feePool.CommunityPool.Add(scraps...)
 		app.DistrKeeper.SetFeePool(ctx, feePool)
 
-		app.DistrKeeper.Hooks().AfterValidatorCreated(ctx, val.GetOperator())
+		err := app.DistrKeeper.Hooks().AfterValidatorCreated(ctx, val.GetOperator())
+		if err != nil {
+			panic(err)
```

Comment:
What is the rationale for panicing here? Can this be documented too?

Reply:
mainly because the function that the `IterateValidators` method relies on as input doesn't handle errors. Also, errors here should only be because of invalid validator data was committed to, which we don't know how to handle anyway.